CONTRIBUTION.md

```diff
@@ -0,0 +1,83 @@
+# Contribution To The Project
+
+Pull Requests and Issues have four possible Labels:
+* WIP - Work In Progress, it is not ready for review/merge
+* Functional - Proposed change changes the test behaviour
+* Metadata - Only Metadata is about to change
+* Scope - Big scope, a commit changes multiple test scenarios
+
+Rules:
+* A commit introduces one logical unit and it should not alter multiple testcases
+* Do not merge your own Pull Request
+
+
+Workflow:
+* Create a fork
+* Clone a fork
+* Add remote to your fork, call it upstream
+```
+$ git remote add upstream git@github.com:RedHat-SP-Security/tests.git
+```
+* Check remote
+```
+$ git remote -v
+# it should print 4 entries e.g.
+origin	git@github.com:radosroka/tests.git (fetch)
+origin	git@github.com:radosroka/tests.git (push)
+upstream	git@github.com:RedHat-SP-Security/tests.git (fetch)
```

Comment:
If you take the previous suggestions of using https://github.com/RedHat-SP-Security/tests.git, please update the expected output here to match it.